cmd/config/subcommand/config/init_flags.go

```diff
@@ -3,7 +3,7 @@ package config
 //go:generate pflags Config --default-var DefaultConfig --bind-default-var
 var (
 	DefaultConfig = &Config{
-		Insecure: true,
+		Insecure: false,
```

Comment:
Can we also document this in flytectl docs where ever these options are referenced

Reply:
Also does this cause any changes to the sandbox config.

Reply:
no